src/steps/os/unix.rs

```diff
@@ -512,6 +512,13 @@ pub fn run_rcm(ctx: &ExecutionContext) -> Result<()> {
     ctx.run_type().execute(rcup).arg("-v").status_checked()
 }
 
+pub fn run_maza(ctx: &ExecutionContext) -> Result<()> {
+    let maza = require("maza")?;
+
+    print_separator("maza");
+    ctx.run_type().execute(maza).arg("update").status_checked()
```

Comment:
Return with ? at the eol and then the functions should return a Ok(())

Reply:
You mean something like this?
```rust
    ctx.run_type().execute(maza).arg("update").status_checked()?;
    Ok(())
}
```
Any reason why we should write it in this way?

Reply:
Could be that I'm mistaken but else if `require()` fails the `Err` doesn't get returned.

Reply:
I am a little bit confused, why would the error derived from `require()` won't be returned? It is followed by a question mark `?`:
```rust
let maza = require("maza")?;
```